### ISSUE TITLE
TypeError: tab is null

### ISSUE THREAD

**Issue by kstev99** [Drive4ik/simple-tab-groups#44]
I have 9 groups, 8 of them I can navigate to just fine, but one group that contains 7 tabs produces an error message (TypeError: tab is null) when i go to it and it brings up 2 blank tabs rather than the 7 tabs assigned to that group.  Since the other 8 groups are working fine, I thought I would delete that group and recreate.  Could not delete the group, since none of the tabs were present in the popup (toolbar icon) and when I choose "Manage Groups" I now get a blank screen other than the text "Simple Tab Groups" in the upper left corner

**Comment by Drive4ik** [comment]
What version you are used?

**Comment by Drive4ik** [comment]
What version you are used?

**Comment by kstev99** [comment]
version 2.2.1

**Comment by kstev99** [comment]
The problem seems to be fixed for now.  The backup/restore function that you added recently is a lifesaver.  Last night I tried restoring a backup and still had the TypeError: tab is nul error.  Today I Uninstalled the add-on and restored a previous backup.  Working fine, something got corrupt somewhere apparently.  Thank you for the great add-on.

**Comment by Drive4ik** [comment]
What other addons do you use?
Maybe it's a conflict with another addon.
In version 2.2.1 the conflict with another addon "New Tab Override" was fixed.
Please, contact me if it's issue replicate again.

**Comment by kstev99** [comment]
Thanks for the support.  I'll let you know if the problem re-appears.  I do use a lot of other addons.  firefox multi-account containers, conex, distill, just to mention a few.  

I first noticed the problem after I checked the box to open group in a NEW window, and then decided that I didn't like that and switched back.. I doubt that it has anything to do with it, but maybe worth mentioning.  

Thanks again for the AWESOME add-on and great support. I'm going to close this issue for now
